chain/stream/types/response.go

```diff
@@ -0,0 +1,75 @@
+package types
+
+import (
+	"fmt"
+)
+
+type StreamResponseMap struct {
+	BlockHeight                          uint64
+	BankBalancesByAccount                map[string][]*BankBalance
+	SpotOrdersBySubaccount               map[string][]*SpotOrder
```

Comment:
SpotOrdersBySubaccount and SpotOrdersByMarketID map[string][]*SpotOrder -> map[string][]*SpotOrderUpdate